server/etcdserver/bootstrap_test.go

```diff
@@ -138,3 +150,165 @@ func mockMembersJSON(m []etcdserverpb.Member) string {
 	members, _ := json.Marshal(m)
 	return string(members)
 }
+
+func TestBootstrapBackend(t *testing.T) {
+	tests := []struct {
+		name                  string
+		prepareData           func(config.ServerConfig) error
+		expectedConsistentIdx uint64
+		hasError              bool
```

Comment:
What's the difference between hasError and expectedError? Could we instead have just `expectedError` that we check `nil` for presence of error?

Reply:
Good point. Thanks & Fixed.